logout.php

```diff
@@ -1,17 +0,0 @@
-<?php require_once('views/head.php'); ?>
-  <title>Logout :: POW! Comic Book Manager</title>
-</head>
-<body>
-  <?php include 'views/header.php';?>
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        Logged out.
-        <br/>
-        <a href="<?php echo filter_input ( INPUT_GET, 'return' ); ?>">Go Back</a>
-     </div>
-    </div>
-  </div>
-  <?php include 'views/footer.php';?>
-</body>
-</html>
```

Comment:
This page is no longer necessary now! Upon logout, the page just goes back to the Index.php and shows a notification.